components/kyma-environment-broker/cmd/cli/skr/options.go

```diff
@@ -0,0 +1,227 @@
+	switch opt {
+	case table, json, yaml:
+		return nil
+	}
+	return fmt.Errorf("invalid value for output: %s", opt)
+}
+
+// SetRuntimeTargetOpts configures runtime target options on the given command
+func SetRuntimeTargetOpts(cmd *cobra.Command, targetInputs *[]string, targetExcludeInputs *[]string) {
+	cmd.Flags().StringArrayVarP(targetInputs, "target", "t", nil,
+		`List of runtime target specifiers to include (the option can be specified multiple times).
+A target specifier is a comma separated list of the following selectors:
+  all                 : all SKRs provisioned successfully and not deprovisioning
+  account=<REGEXP>    : Regex pattern to match against the runtime's GlobalAccount field. E.g. CA50125541TID000000000741207136, "CA.*"
+  subaccount=<REGEXP> : Regex pattern to match against the runtime's SubAccount field. E.g. 0d20e315-d0b4-48a2-9512-49bc8eb03cd1
+  region=<REGEXP>     : Regex pattern to match against the shoot cluster's Region field (not SCP platform-region). E.g. "europe|eu-"
+  runtime-id=<ID>     : Runtime ID is used to indicate a specific runtime`)
+	cmd.Flags().StringArrayVarP(targetExcludeInputs, "target-exclude", "e", nil,
+		`List of runtime target specifiers to exclude (the option can be specified multiple times).
+A target specifier is a comma separated list of the selectors described under --target option`)
+}
+
+// ValidateTransformRuntimeTargetOpts checks the validity of runtime target options, and transforms them for internal usage
+func ValidateTransformRuntimeTargetOpts(targetInputs []string, targetExcludeInputs []string, targetSpec *internal.TargetSpec) error {
+	if len(targetInputs) == 0 {
+		return errors.New("at least one runtime target must be specified with --target")
+	}
+	for _, target := range targetInputs {
+		err := parseRuntimeTarget(target, &targetSpec.Include, true)
+		if err != nil {
+			return err
+		}
+	}
+	for _, target := range targetExcludeInputs {
+		err := parseRuntimeTarget(target, &targetSpec.Exclude, false)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func parseRuntimeTarget(targetInput string, targets *[]internal.RuntimeTarget, include bool) error {
+	target := internal.RuntimeTarget{}
+	selectors := strings.Split(targetInput, ",")
+	var flagName string
+	if include {
+		flagName = "--target"
+	} else {
+		flagName = "--target-exclude"
+	}
+
+	for _, selector := range selectors {
+		sv := strings.Split(selector, "=")
+		selectorKey := sv[0]
+		var selectorValue string
+		if len(sv) > 1 {
+			selectorValue = sv[1]
+		} else {
+			selectorValue = ""
+		}
+		switch selectorKey {
+		case internal.TargetAll:
+			if !include {
+				return errors.New("\"all\" cannot be used with --target-exclude")
+			}
+			target.Target = internal.TargetAll
+		case "account":
+			err := checkRuntimeTargetSelector(selectorKey, selectorValue, flagName)
+			if err != nil {
+				return err
+			}
+			target.GlobalAccount = selectorValue
+		case "subaccount":
```

Comment:
I think we can export these selectors as const and use them on the package level

Reply:
Done